lib/src/crypto/hmac_drbg_secure_random.dart

```diff
@@ -0,0 +1,65 @@
+import 'dart:typed_data';
+
+import 'package:hash/hash.dart';
+import 'package:pointycastle/export.dart';
+
+import '../utils.dart';
+import '../utils/hmac_drbg.dart';
+
+class HmacDrbgSecureRandom implements SecureRandom {
+  HmacDRBG _hmacDRBG;
+
+  HmacDrbgSecureRandom()
+      : _hmacDRBG = HmacDRBG(
+            hash: SHA256(),
+            entropy: Uint8List.fromList([0]),
+            nonce: null,
+            pers: null,
+            outLen: 256);
+
+  @override
+  String get algorithmName => 'HmacDrgbSecureRandom';
+
+  @override
+  BigInt nextBigInteger(int bitLength) {
+    // This was made to match jsbn.js's behavior of generating 1 extra byte
```

Comment:
Exactly :D We had a really hard time last time we tried to implement this because we couldn't find this 1 extra byte difference from our implementation :)